content/introduction/_index.md

```diff
@@ -96,10 +96,10 @@ The official charter for SODA Foundation under Linux Foundation can be found [he
 Please join us to collaborate, research, develop and lead for unified data framework and standards.
 Contact us through any of the following channels:
 
- - [SODA Website](https://sodafoundation.io/the-foundation/join/)  
- - [SODA Slack](https://sodafoundation.io/slack)
- - [SODA Twitter](https://twitter.com/sodafoundation)  
- - [SODA Mailing List](https://lists.sodafoundation.io)
- - [SODA LinkedIn](https://www.linkedin.com/company/sodafoundation/)
+ - <img src="https://github.com/FortAwesome/Font-Awesome/blob/6.x/svgs/brands/github.svg" width="20" height="20">&nbsp; [SODA Website](https://sodafoundation.io/the-foundation/join/)  
```

Comment:
Please check the deploy preview.
The logos are not appearing.  
![image](https://user-images.githubusercontent.com/19162717/229417355-8e416e9b-d03d-4093-b8ef-2552e8cc8b07.png)

Reply:
sure sir...I will check and update it